homeassistant/components/music_assistant/quality_scale.yaml

```diff
@@ -0,0 +1,70 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: exempt
+    comment: Integration is local push
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
It's not required to have `await hass.async_block_till_done()` after doing a config flow step as those generally do not fire events